ci/build_wheel.sh

```diff
@@ -8,4 +8,4 @@ mkdir -p ./dist
 python -m pip wheel . --wheel-dir=./dist -vvv --disable-pip-version-check --no-deps
 
 rapids-logger "Upload Wheel"
-RAPIDS_PY_WHEEL_NAME="numba_cuda" rapids-upload-wheels-to-s3 ./dist
+RAPIDS_PY_WHEEL_NAME="numba-cuda" rapids-upload-wheels-to-s3 ./dist
```

Comment:
I believe that this needs to stay with an underscore. This gets used for the wheel filename, which is always normalized to have underscores even if the package name has a hyphen IIRC.

Reply:
Thanks, I've undone this now.